packages/frontend/src/pages/Template/InvalidTemplatePage.tsx

```diff
@@ -0,0 +1,12 @@
+import { Stack, Text } from '@chakra-ui/react'
+
+export default function InvalidTemplatePage() {
+  // TODO (mal): check if this needs to be beautified
+  return (
+    <Stack mt={24} justifyContent="center" alignItems="center" gap={2}>
+      <Text textStyle="subhead-1">
+        Template not found, please refer to our templates page!
```

Comment:
```suggestion
        Template not found, please refer to our <link>templates</link> page!
```
maybe link back to templates page

Reply:
im going to remove this page and only allow users to load the template modal if it is valid, so `templates/abc` will just remain on the same page